OneCommunityGlobal/HighestGoodNetworkApp
# Shubhankar added missing box shadow style to buttons

# Description
(PRIORITY LOW) Jae: Add Profile Page missing box Shadow Style in User Profile Component

## Related PRS (if any):
…

## Main changes explained:
- Update UserProfile.jsx with box styles.
- Update BadgeReport.jsx with box styles.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user
4. go to User Profile. 
5. Verify the following:
    - View the profile in mobile view and check if the modal buttons (below the badge section) have box shadow. 
    - Go to a user's profile who has an assigned badge, click on `Select Featured` to view Full Badge History View and check if the Delete button has a box shadow.
    -  Go to a user's profile and check whether the `Generate Summary Intro` button has a box shadow. 
## Screenshots or videos of changes:
![Screenshot 2023-11-11 114037](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59073366/d0103d62-0977-4d2b-ae69-d4f84c453392)
![Screenshot 2023-11-11 115639](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59073366/ebca821a-9ad2-40e9-8727-e64af1ba2af0)
![Screenshot 2023-11-11 115817](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59073366/813b2fe6-cdff-47a5-ae98-4b61d1bb465d)

## Note:
Please check other buttons in User Profile page which may require box shadow.
Disregard the change in Image.test.jsx.
